docs/sphinx/sections/api/apidocs/cli.rst

```diff
@@ -7,6 +7,10 @@ Dagster CLI
    :prog: dagster asset
    :nested:
 
+.. click:: dagster._cli.code_server:code_server_cli
```

Comment:
I don't see the new section reflected in the API docs, just the original `dagster api grpc`